src/Command/ImportDefaultDataCommand.php

```diff
@@ -0,0 +1,130 @@
+use App\Repository\CurriculumInventoryInstitutionRepository;
+use App\Repository\LearningMaterialStatusRepository;
+use App\Repository\LearningMaterialUserRoleRepository;
+use App\Repository\MeshDescriptorRepository;
+use App\Repository\SchoolRepository;
+use App\Repository\SessionTypeRepository;
+use App\Repository\TermRepository;
+use App\Repository\UserRoleRepository;
+use App\Repository\VocabularyRepository;
+use App\Service\DefaultDataLoader;
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\LockableTrait;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ImportDefaultDataCommand extends Command
+{
+    use LockableTrait;
+
+    public function __construct(
+        protected DefaultDataLoader $dataLoader,
+        protected AamcMethodRepository $aamcMethodRepository,
+        protected AamcPcrsRepository $aamcPcrsRepository,
+        protected AamcResourceTypeRepository $aamcResourceTypeRepository,
+        protected AlertChangeTypeRepository $alertChangeTypeRepository,
+        protected ApplicationConfigRepository $applicationConfigRepository,
+        protected AssessmentOptionRepository $assessmentOptionRepository,
+        protected CompetencyRepository $competencyRepository,
+        protected CourseClerkshipTypeRepository $courseClerkshipTypeRepository,
+        protected CurriculumInventoryInstitutionRepository $curriculumInventoryInstitutionRepository,
+        protected LearningMaterialStatusRepository $learningMaterialStatusRepository,
+        protected LearningMaterialUserRoleRepository $learningMaterialUserRoleRepository,
+        protected MeshDescriptorRepository $meshDescriptorRepository,
+        protected SchoolRepository $schoolRepository,
+        protected SessionTypeRepository $sessionTypeRepository,
+        protected TermRepository $termRepository,
+        protected UserRoleRepository $userRoleRepository,
+        protected VocabularyRepository $vocabularyRepository
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('ilios:import-default-data')
+            ->setAliases(['ilios:setup:import-default-data'])
+            ->setDescription('Imports default application data into Ilios. Only works with an empty database schema.');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$this->lock()) {
+            $io->error('The command is already running in another process.');
+            return 0;
+        }
+
+        $io->warning('Do not run this against an Ilios instance that already contains data!');
```

Comment:
we can't. the `application_config` table will contain data regardless.
but maybe it would be enough to just peek into the `school` table - that one should be empty under any circumstances.

Reply:
:heavy_check_mark: